app/models/ldap_auth_source.rb

```diff
@@ -101,10 +101,13 @@ def get_user_attributes_from_ldap_entry(entry)
     }
   end
 
-  # Return the attributes needed for the LDAP search.  It will only
-  # include the user attributes if on-the-fly registration is enabled
-  def search_attributes
-    if onthefly_register?
+  # Return the attributes needed for the LDAP search.
+  #
+  # @param [Boolean] all_attributes Whether to return all user attributes
```

Comment:
You have to swap `[Boolean]` and `all_attributes`! The name comes first, then the type.

Reply:
Oops!